ci-operator/config/devfile/integration-tests/devfile-integration-tests-master.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-devfile
+  commands: ls -la
+  openshift_installer_src:
+    cluster_profile: aws
```

Comment:
1. To run e2e tests you're missing one of the stanzas that define which OCP release will be used to install the testing cluster (see https://docs.ci.openshift.org/docs/architecture/ci-operator/#describing-inclusion-in-an-openshift-release)
2. Template based tests are obsolete and you should not add new ones. Use multi-stage tests instead (https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration)

Reply:
Thanks, @petr-muller for your guidance. As a newbie for 'ci-operator',  I actually ran 'make new-repo' command found from [here](https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#bootstrapping-configuration-for-a-new-repository) and it generated the files for this PR. I think the makefile may be outdated. Anyway, I updated the files according to your suggestion.

Reply:
That's a **great** point! We're in the process of deprecating templates and we will definitely need to update the tool to generate the new content. Thanks for pointing this out!